css/styles.css

```diff
@@ -993,3 +993,76 @@ nav .btn {
 		margin-right: 0;
 	}
 }
+
+.customer__cards {
+        display: flex;
+        height: 30rem;
+        max-width: 92rem;
+      
+    }
```

Comment:
can we clean up the formatting to match above? indentation looks fine up until this line when everything gets shifted with a 4 space indent.